src/Core/AdminConsole/Services/Implementations/PolicyService.cs

```diff
@@ -63,19 +67,24 @@ public PolicyService(
         _savePolicyCommand = savePolicyCommand;
         _removeOrganizationUserCommand = removeOrganizationUserCommand;
         _organizationHasVerifiedDomainsQuery = organizationHasVerifiedDomainsQuery;
+        _currentContext = currentContext;
     }
 
-    public async Task SaveAsync(Policy policy, Guid? savingUserId)
+    public async Task SaveAsync(Policy policy, Guid? savingUserId, EventSystemUser? eventSystemUser = null)
     {
         if (_featureService.IsEnabled(FeatureFlagKeys.Pm13322AddPolicyDefinitions))
         {
             // Transitional mapping - this will be moved to callers once the feature flag is removed
+            // TODO make sure to populate with SystemUser if not an actual user
```

Comment:
TODO no longer needed?

Reply:
This TODO will get addressed with a subsequent story.

Reply:
I can do this when I remove the policy definition feature flag shortly. Callers will have to construct their own `PolicyUpdate` objects, so they'll need to make sure they're specifying the right actingUser.